solr/core/src/test/org/apache/solr/analysis/TestWordDelimiterGraphFilterFactory.java

```diff
@@ -207,16 +189,16 @@ public void testCustomTypes() throws Exception {
     args.put("splitOnCaseChange", "1");
     
     /* default behavior */
-    WordDelimiterFilterFactory factoryDefault = new WordDelimiterFilterFactory(args);
+    WordDelimiterGraphFilterFactory factoryDefault = new WordDelimiterGraphFilterFactory(args);
     factoryDefault.inform(loader);
     
     TokenStream ts = factoryDefault.create(whitespaceMockTokenizer(testText));
     BaseTokenStreamTestCase.assertTokenStreamContents(ts, 
-        new String[] { "I", "borrowed", "5", "540000", "400", "00", "at", "25", "interest", "interestrate", "rate" });
+        new String[] { "I", "borrowed", "540000", "5", "400", "00", "at", "25", "interestrate", "interest", "rate" });
```

Comment:
Slightly concerning here that we are dependent on order from something that I presume is not order-defined.